src/components/biblio/topic_entity_tag/TopicEntityTable.js

```diff
@@ -225,40 +245,34 @@ const TopicEntityTable = () => {
   const headersToEntityMap = new Set(['topic', 'entity_type', 'entity', 'display_tag']);
   const headerToLabelMap = { 'negated': 'no data', 'novel_topic_data': 'novel data' };
 
-
-  // TODO: use the following code for the 'simple' table
-  // for (const tetDict of topicEntityTags.values()) {
-  //   for (const tetDictKey in tetDict) {
-  //     // console.log(tetDictKey);
-  //     if (tetDictKey === 'topic_entity_tag_source') {
-  //       for (const tetSourceKey in tetDict[tetDictKey]) {
-  //         if ( (source_headers.indexOf(tetSourceKey) === -1) && !(excludeColumnSet.has(tetSourceKey)) ) { source_headers.push(tetSourceKey); } } }
-  //     else {
-  //       if ( (headers.indexOf(tetDictKey) === -1) && !(excludeColumnSet.has(tetDictKey)) ) { headers.push(tetDictKey); } }
-  //   }
-  // }
-
-   
   return (
-      <div>
-        <LoadingOverlay active={isLoadingData || isLoadingMappings} />
```

Comment:
I think line 244 and 245 should stay in

Reply:
Good catch! Added back the missing line. Thanks @azurebrd